File: BikeManagerAPI/BikeManagerAPI/Models/ItemOrdemServico.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace BikeManagerAPI.Models
{
    public partial class ItemOrdemServico
    {
        public int CdProduto { get; set; }
        public int CdOrdemServico { get; set; }
        public int CdTabelaPreco { get; set; }
        public decimal VlUnitario { get; set; }
        public decimal VlTotal { get; set; }
        public int QtProduto { get; set; }
        public string DsObservacao { get; set; }
        public DateTime DtRegistro { get; set; }
        public DateTime? DtAlteracao { get; set; }
        public OrdemServico OrdemServico { get; set; }
        public Produto Produto { get; set; }
        public TabelaPreco TabelaPreco { get; set; }
    }
}

File: WebApp/src/app/cadastros/models/itemOrdemServico.ts
import { Produto } from "./produto";
import { OrdemServico } from "./ordemServico";
import { TabelaPreco } from "./tabelaPreco";

export class ItemOrdemServico {
    cdProduto: number;
    cdOrdemServico: number;
    cdTabelaPreco: number;
    vlUnitario: number;
    vlTotal: number;
    qtProduto: number;
    vlCusto: number;
    vlCustoMedio: number;
    dsObservacao: string;
    dtRegistro: Date;
    dtAlteracao: Date;
    ordemServico: OrdemServico;
    produto: Produto;
    tabelaPreco: TabelaPreco;
    countItem: number;
    tpItem: string;
}